### ISSUE TITLE
May I please refer to your design in my project?

### ISSUE THREAD

**Issue by ASRT9** [KRALYN/PositronV3#39]
May I please refer to your design in my project?

**Comment by RCPlayland** [comment]
Yes you may.  Also be sure to include the original designer as well.  His link is in my description
    On Saturday, March 25, 2023 at 09:11:16 AM CDT, ASRT9 ***@***.***> wrote:  
 
 


May I please refer to your design in my project.

**Comment by RCPlayland** [comment]
That is if it is a re-design.

I do have a couple originals.

Which design is it that you are referring to?
    On Saturday, March 25, 2023 at 09:11:16 AM CDT, ASRT9 ***@***.***> wrote:  
 
 


May I please refer to your design in my project.

**Comment by nomadsgalaxy** [comment]
Hello, I know this is late, but yes - this is open source, so you can always reference this as long as you follow the creative commons licensing.

Please see the latest version of the project here; https://github.com/Positron3D/Positron
